src/nvim/gettext.h

```diff
@@ -11,6 +11,9 @@
 #  define N_(x) x
 # endif
 # define NGETTEXT(x, xs, n) ngettext(x, xs, n)
+# if defined(__APPLE__) && defined(setlocale)
```

Comment:
worth a comment here mentioning `#9789`, and linking to the postgresql thread or some other link you found.

Reply:
Hmm, I can't find a link that I really like. The Postgres one is meant for WIN32.

Would be this comment good enough?

```
// On a Mac, gettext's libintl.h defines "setlocale" to be replaced by
// "libintl_setlocal" which leads to wrong return values. #9789
```

Reply:
ok.